templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,51 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
```

Comment:
Delete line 7. `{!go-to-the.md!} [Your Bots](/#settings/your-bots)` and `{!settings.md!}` need to be on separate lines.